### PR TITLE
fix(DB/Account): fixed CHARSET in account_muted

### PR DESCRIPTION
**Pull request by r0m1ntik**
## Changes Proposed:
-  Changed in account_muted CHARSET to utf8

## Issues Addressed:
- Closes [4213](https://github.com/azerothcore/azerothcore-wotlk/issues/4213)

## Tests Performed:
Снимок экрана 2021-01-05 в 14 56 42

## How to Test the Changes:

1) apply sql
2) add in account_muted in column mutedby or mutereason Chinese text

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by mpfans** [comment]
account_ Muted tested to confirm that Chinese can be input
In the same way, logs can also be input in Chinese,but must run the following SQL statement

ALTER TABLE `logs`
CHARSET = 'utf8',
MODIFY `string` TEXT(255) NOT NULL;

**Comment by Helias** [comment]
@r0m1ntik edit your sql code to make this work also for chinese  as @mpfans suggests 👍

File: data/sql/updates/pending_db_auth/rev_1609867708436603000.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_auth` (`sql_rev`) VALUES ('1609867708436603000');
+ALTER TABLE `account_muted`
+CHARSET = 'utf8',
+MODIFY `mutedby` varchar(50) NOT NULL,
+MODIFY `mutereason` varchar(255) NOT NULL;
+ALTER TABLE `logs`
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_auth/rev_1609867708436603000.sql:6]: missing
CHARSET = 'utf8',

**Comment by Si1ker** [comment]
@r0m1ntik 👀  ?

**Comment by r0m1ntik** [comment]
@Si1ker done :)

**Pull request merged**